recipes/tl-expected/all/test_package/conanfile.py

```diff
@@ -3,7 +3,7 @@
 
 class ExpectedTestConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
-    generators = "cmake"
+    generators = "cmake_find_package"
```

Comment:
Please always keep `cmake` in the generators of `test_package`.
The `cmake` generator ensures that the compiler is configured according to the profile. 
```suggestion
    generators = "cmake", "cmake_find_package"
```